sympy/concrete/summations.py

```diff
@@ -254,6 +254,28 @@ def _eval_derivative(self, x):
         else:
             return NotImplementedError('Lower and upper bound expected.')
 
+    def _eval_integral(self, x):
+        if isinstance(x, Symbol) and x not in self.free_symbols:
+            return x*self
+
+        f, limits = self.function, list(self.limits)
+
+        limit = limits.pop(-1)
+
+        if limits:
```

Comment:
Yes  this is tested ,actually limits returns an array containing tuples which in turn contain limits of summation "Some thing like this [(limits of the summation)]", so when we use limits.pop(-1) we get the tuple "(limits of summation)",Which is later used to extract each limits separately